src/components/ThoughtBox.jsx

```diff
@@ -0,0 +1,36 @@
+import { LikeButton } from "./LikeButton";
+import PropTypes from "prop-types";
+import "./ThoughtBox.css";
+
+export const ThoughtBox = ({ thoughts, setThoughts }) => {
+  return (
+    <ul>
+      {thoughts.map((thought) => {
+        //Calculation how many minutes have passed since the post
+        const createdAt = new Date(thought.createdAt);
+        const currentTime = new Date();
+        const timeDifference = Math.floor(
+          (currentTime - createdAt) / (1000 * 60)
```

Comment:
I saw that this results in a time display that only shows minutes ago, meaning it says ex 1678 minutes ago when its a bit older.. It would be great if it turned into hours and days etc. I assume you heard of the recommended formatDistance that you can import from 'date-fns'? It does it automatically :D